main.js

```diff
@@ -78,16 +114,11 @@ function HandleNode(svgNode, scaleX, scaleY, translateX, translateY) {
           break;
         }
 
-        return HandleNode(svgElement, scaleX, scaleY, translateX, translateY);
+        return HandleNode(svgElement, scaleX, scaleY, translateX, translateY, preserveFill);
 
       // PATH ------------------------------------------------------------------
       case 'path':
-        var isStrokePath = svgElement.getAttribute('stroke') &&
-                           svgElement.getAttribute('stroke') != 'none';
-        // If fill is none and doesn't have stroke, this is probably one of those worthless paths
-        // of the form <path fill="none" d="M0 0h24v24H0z"/>
-        if (svgElement.getAttribute('fill') == 'none' && !isStrokePath)
```

Comment:
Removed this and moved it below since the check for `svgElement.getAttribute('fill') == 'none'` is handled in GetPathColorCommandFromFill

Reply:
I don't think you've preserved the behavior. The intent is to completely drop `<path fill="none" d="M0 0h24v24H0z"/>`, which is why there's a `break`.

Reply:
fair, added back